core/worker.h

```diff
@@ -99,8 +97,8 @@ class Worker {
  private:
   volatile worker_status_t status_;
 
-  int wid_;  /* always [0, MAX_WORKERS - 1] */
-  int core_; /* TODO: should be cpuset_t */
+  int wid_;   // always [0, kMaxWorkers - 1]
+  int core_;  // TODO: should be cpuset_t
```

Comment:
It is about having a set of CPU cores (not only one) on which a worker thread can be scheduled by OS (if the set has just one core, the behavior is same as before). Since it would require many changes here and there, let's do it sometime later in a separate PR.